src/RoboFile/RoboFileInterface.php

```diff
@@ -43,4 +43,26 @@ public function runTestSuites(
 	 * @since   3.7.0
 	 */
 	public function prepareTestingPackage($params = array('dev' => false));
+	/**
+	 * Function for actual execution of a individual test of this extension
+	 *
+	 * @param   array  $opts  Array of configuration options:
+	 *                        - 'env': set a specific environment to get configuration from
+	 *                        - 'debug': executes codeception tasks with extended debug
+	 * 						  - 'suite':
+	 * 						  - 'test': example: 'install/InstallWeblinksCest.php/installWeblinks'
+	 *
+	 * @return void
+	 *
+	 * @since   x.x.x
+	 */
+	public function runTestSingle(
```

Comment:
The reason why I don't think `runTestSingle` should be in the interface is because the function will actually be the same for every extension, so I think it's a task that should be in the class itself, and should not (in most occasions) be overridden in a child implementation of an specific extension